packages/core/src/fetchers/fetchExperienceEntry.ts

```diff
@@ -1,48 +1,58 @@
 import { ContentfulClientApi, Entry } from 'contentful';
 
-type fetchExperienceEntryArgs = {
-  client: ContentfulClientApi<undefined>;
+type ClientAndLocaleParams =
+  | {
+      client: ContentfulClientApi<undefined>;
+      locale?: string;
```

Comment:
```suggestion
      locale: string;
```
Does this need to be required in this case?